Build/gcc/Makefile

```diff
@@ -88,8 +88,9 @@ INCLUDE_FLAGS = -I$(GENERICS_DIR) \
 # All flags to be used in compilation. We're not complying with C++98 yet due
 # to the hostlink and some other minor changes, though nobody has studied this
 # comprehensively.
+# Trivial log handler is defined for now - we need to think about this more.
```

Comment:
Why do we need to think about this more?

Reply:
The trivial log handler is very much a "it just works for getting messages out". Implementation everywhere (inc. in softswitch) is rudimentary, but the whole log message needs a look at.